canonical-web-and-design/charmhub.io
# Fix empty link errors

## Done

Resolved a number of low hanging accessibility issues, all relating to empty links.

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- If you don't have it already, install the [WAVE extension]() for Chrome or Firefox.
- Launch the extension, see that the number of errors is zero, and compare to https://charmhub.io/, where there are 17 errors.
  - 10 of these errors relate to placeholder elements that all linked to the homepage, but they seem to only be made visible when data is taking a while to load, and don't need to link anywhere.
